vaadinfrontend/frontend/themes/datamanager/components/page-area.css

```diff
@@ -0,0 +1,35 @@
+/* All custom css for components which extend the page-area component go in here! */
+
+.page-area {
+  background-color: var(--lumo-base-color);
+  border: 1px solid;
+  border-color: var(--lumo-contrast-10pct);
+  border-radius: var(--lumo-border-radius-m);
+  box-shadow: var(--lumo-box-shadow-m);
+  box-sizing: border-box;
+  text-overflow: ellipsis;
+  max-width: 100%;
+  max-height: 100%;
+  margin: var(--lumo-space-s);
+  padding: var(--lumo-space-l);
+}
+
+.attachments-area {
+  display: flex;
+  gap: var(--lumo-space-m);
+  flex-direction: column;
+}
+
+.attachments-area .title {
+  font-weight: bold;
+  color: var(--lumo-secondary-text-color);
+  font-size: var(--lumo-font-size-xxl);
+  margin-bottom: 0.5rem;
+}
+
+.attachments-area .delete-button {
+  color: var(--lumo-error-text-color);
+  --lumo-button-size: var(--lumo-size-m);
+  cursor: pointer;
+  background-color: inherit;
```

Comment:
Why do you inherit the background color? As the initial value for background-color is `transparent` this should not be necessary?

Reply:
This is from the CSS created by adding the different button settings with Lumo. If I remove this setting it looks like this:
<img width="438" alt="image" src="https://github.com/qbicsoftware/data-manager-app/assets/10232219/0e916748-029a-4350-ba96-6f35a029c7b4">
So the default value seems to be gray - I think it's the button background?